chapters/introduction.tex

```diff
@@ -22,6 +22,11 @@ \section{Scope of the Specification}\label{scope-of-the-specification}
 verified for those classes, before using them to build the simulation
 model.
 
+The restrictions for a simulation model apply when simulating the model.
+Thus a tool may perform parts of the translation of a model lacking some key parts -- provided they are given when simulating the model.
+If the tool has some special mechanism for adjusting the model (like setting parameters), it is only necessary that requirements for a simulation model are satisfied for this adjusted model when simulating it.
+The specification currently uses the term "translation" for this combination of compilation and adjustment, regardless of whether the adjustment happen before or after the compilation.
```

Comment:
For me, this does not match what I have in mind, and a language group discussion would be good before making such a definition.

I think it should also be noted that to make this work, a tool needs to perform quite a bit of translation in order to even determine what the set of parameters without value is.  The way I imagine it would work in the end, one would perform a complete translation without the parameter values, and the result of the translation then needs to represent which of the parameters that must be given values after translation.  Avoiding the need for full translation seems doable, but impractical and of little value.

Reply:
A very minor suggestion on the markup, just to get the markup concerns out of the way:
```suggestion
The specification currently uses the term \emph{translation} for this combination of compilation and adjustment, regardless of whether the adjustment happen before or after the compilation.
```